prometheus/handlers/handlers.go

```diff
@@ -256,6 +267,7 @@ func decodeRulePostRequest(c echo.Context) (rulefmt.Rule, error) {
 	jsonPayload := alert.RuleJSONWrapper{}
 	err = json.Unmarshal(body, &jsonPayload)
 	if err != nil {
+		glog.Errorf("Error decoding rule: %v", err)
```

Comment:
This is the same message as 257, can we log something to differentiate the two for greps?